### DISCUSSION TITLE
Report Issue - Incorrect values populated when having failures

### DISCUSSION THREAD

**Discussion by sgolyala1** [test-results-reporter/reporter#97] [Q&A]
@ASaiAnudeep I see the issue with logging incorrect number of failures - please can you look into this asap
I'm expecting only 1 failure and I see 2 showing in the result
![image](https://user-images.githubusercontent.com/78226609/195131467-e5daa301-580c-4f47-97d2-d502ef16edb1.png)
[junitReport.txt](https://github.com/test-results-reporter/reporter/files/9757462/junitReport.txt)

**Comment by ASaiAnudeep** [comment]
It seems to be a bug from the testing tool you are using here. As I can see, the following test suite has an attribute of **failures** with value **2**. Did the test run had retries by any chance?

```xml
<testsuite name="Insights / leasecompslisting" tests="3" failures="2" errors="0">
    <testcase name="Lease comps Status  is 200" time="8.324" classname="MainApi"/>
    <testcase name="Lease comps listing page -Table - response time is less than 5s" time="8.324" classname="MainApi">
      <failure type="AssertionFailure" message="expected 8324 to be below 5000">
    
      </failure>
    </testcase>
    <testcase name="Verify Leasecomps  listing page table rows > 80k" time="8.324" classname="MainApi"/>
  </testsuite>
  ```

**Reply by sgolyala1** [reply]
@ASaiAnudeep Thankyou for looking into this.I don't have anywhere setting up the retries and think raised it with postman/newman as well as think it's xml report(newman) which is populating this value [https://github.com/postmanlabs/newman/discussions/3024]
